guides/common/modules/proc_invalidating-jwts-of-all-users.adoc

```diff
@@ -0,0 +1,33 @@
+[id="invalidating-jwts-of-all-users"]
+= Invalidating JWTs of all users
+
+You can invalidate all registration JSON Web Tokens of all users at once.
+
+To use the CLI instead of the {ProjectWebUI}, see the xref:cli-invalidating-jwts-of-all-users[].
+To use the API, see the xref:api-invalidating-jwts-of-all-users[].
+
+.Prerequisites
+* Your {Project} user has a role that grants the `view_users` and `edit_users` permissions.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Administer* > *Users*.
+. Click *Invalidate JWTs for all users*.
+. In the confirmation window, click *Confirm*.
+
+.Verification
+* The {ProjectWebUI} displays the following message: `Successfully invalidated registration tokens for all users.`
+
+[id="cli-invalidating-jwts-of-all-users"]
+.CLI procedure
+* Invalidate all registration tokens of all users by running Hammer:
++
+[options="nowrap" subs="+quotes,attributes,verbatim"]
+----
+$ hammer registration-tokens invalidate ???
```

Comment:
@girijaasoni What is the command to invalidate tokens for all users?

Reply:
it is not possible to invalidate all jwts at once from CLI Or API as its a destructive action, user will have to manually mention each of the user ID.
It is only possible from UI to do so